lesson01/style.css

```diff
@@ -0,0 +1,12 @@
+h1{
+  padding: 0.25em;
+  font-family: sans-serif;
+}
+.non-h1-body{
+  padding: 0.5em;
+  font-family: sans-serif;
+}
+#array_one, #array_two, #array_three, #array_four{
```

Comment:
If you wanted to be fancy, this selector could be for all elements with an id that starts with "array_": 

```
[id^="array_"] {  }
```

https://www.w3schools.com/css/css_attribute_selectors.asp

Reply:
That is a wonderful suggestion. I am an advocate for simplicity!